lib/state.js

```diff
@@ -157,7 +164,7 @@ StateResolver.prototype.capture_ = function() {
                                     result.error, true)
         };
       } else {
-        evaluated = that.resolveVariable_(expression, result.mirror);
+        evaluated = that.resolveVariable_(expression, result.mirror, true);
```

Comment:
This boolean seems to propagate parallel information with the `isEvaluated` property above. I would be in favor of using the boolean parameter alone to avoid modifying the V8 object.